game/script.js

```diff
@@ -45,70 +46,25 @@ document.addEventListener('keydown',(e)=>{
     console.log(player);
 });
 
-// document.addEventListener('keydown',(e)=>{
-//   const player = document.querySelector(".player");
-//   const index = [...player.parentElement.children].indexOf(player);
-//   switch(e.keyCode){
-//     case 37:
-//       const prev = player.previousElementSibling;
-//       if(!prev || prev.classList.contains("map-wall")){return
-//       }
-//       prev.classList.add("player");
-//       player.classList.remove('player');
-    
-//       break;
-//     case 39:
-//      const next = player.nextElementSibling
-//         if(!next || next.classList.contains("map-wall")){return}
-//       next.classList.add("player"); 
-//       player.classList.remove('player');
-      
-//       break;
-//     case 38:
-//       const upwardsElement = player.parentElement.previousElementSibling.querySelector(`td:nth-child(${index + 1})`);
-//       if(!upwardsElement || upwardsElement.classList.contains("map-wall")){return}
-//       upwardsElement.classList.add('player');
-//       player.classList.remove('player');
-//           break;
-//     case 40: 
-//       const downwardElement = player.parentElement.nextElementSibling.querySelector(`td:nth-child(${index + 1})`);
-//       if(!downwardElement || downwardElement.classList.contains("map-wall")){return}
-//       downwardElement.classList.add('player');
-//       player.classList.remove('player');
-    
-//           break;
-//   };
-// });
-  
-
-
-
+let life = 100;
+let lifePortion = 1;
 
+function changeLife () {
```

Comment:
I think lifePortion should be a param for `changeLife` function
changeLife is not the best name because you can not add life here , it should be name decreseLife